app/Repositories/AudioRepository.php

```diff
@@ -18,10 +18,17 @@ public function __construct(Audio $audio)
     public function getFormFields(TwillModelContract $audio): array
     {
         $fields = parent::getFormFields($audio);
+        $fields['title'] = $audio->title ?? $audio->getApiModel()?->title;
         $fields['selector_number'] = $audio->selector?->number;
         return $fields;
     }
 
+    public function prepareFieldsBeforeSave($audio, $fields): array
+    {
+        $fields['title'] = $fields['title'] != $audio->getApiModel()?->title ? $fields['title'] : null;
+        return parent::prepareFieldsBeforeSave($audio, $fields);
+    }
+
```

Comment:
If you implement the code mentioned above you could omit this and just leave the local title blank.